src/C4MessageInput.cpp

```diff
@@ -649,6 +651,22 @@ bool C4MessageInput::ProcessCommand(const char *szCommand)
 		return true;
 	}
 
+	// mute
+	if (SEqual(szCmdName, "mute"))
+	{
+		if (auto *client = Game.Clients.getClientByName(pCmdPar))
```

Comment:
This creates a warning - I recommend changing it to `if (auto *client = Game.Clients.getClientByName(pCmdPar); client)`